examples/prism-keyman.html

```diff
@@ -0,0 +1,101 @@
+<h1>Keyman</h1>
+<p>To use this language, use the class "language-keyman".</p>
+
+<h2>Comments</h2>
+<pre><code>c This is a comment</code></pre>
+
+<h2>Strings, numbers and characters</h2>
+<pre><code>"'this' is a string"
+'and so is "this"'
+U+0041 d65 x41   c these are all the letter A
+</code></pre>
+
```

Comment:
You should add small examples for virtual keys and prefixes, since there don't seem to be any in the large example below.